servers/javascript/src/conflictResolvers.ts

```diff
@@ -1,6 +1,7 @@
 import type * as cbl from "@couchbase/lite-js";
 import type * as tdk from "./tdkSchema";
-import { check, isDict } from "./utils";
+import {check, isObject} from "./utils";
+import type {JSONObject} from "@couchbase/lite-js";
```

Comment:
type * is imported above, so this can be cbl.JSONObject without this import.  Where is it used though?